### ISSUE TITLE
Add ignore: "blockless-group" option to at-rule-empty-line-before

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#1413]
Ref: https://github.com/stylelint/stylelint/issues/1377 & https://github.com/stylelint/stylelint/issues/1366

**Comment by jeddy3** [comment]
@Nirazul Please consider contributing this option if you require it for your code style.

**Comment by Nirazul** [comment]
Thx, I will look into it!

**Comment by Nirazul** [comment]
@jeddy3 
When testing against following option: `[ "never", { ignore: ["blockless-group"] } ]` I've come across the following code example which is accepted but should be rejected:

```css
@import 'x.css';
 /* <-- here is a whitespace char */
@media {};
```

Is this considered a non-empty line?

**Comment by jeddy3** [comment]
> Is this considered a **non**-empty line?

Yes, I think so. We consider an empty line to be [a sequence of 2 newlines characters](https://github.com/stylelint/stylelint/blob/master/src/rules/at-rule-empty-line-before/index.js#L45-L49).

So, a single whitespace char should trigger a warning if the option is `"never"`. If this isn't the case, can you open a new issue for it?

**Comment by Nirazul** [comment]
As the used option is formulated as following on the stylelint page: 
> There must never be an empty line before at-rules.

The rule works as desired. A line with one whitespace character on it is not considered an empty line.

**Comment by jeddy3** [comment]
Oops, my bad. I got myself confused as I was thinking of the `"always"` option.

> The rule works as desired.

Yep, you're right.
